text/0000-abort-by-default.md

```diff
@@ -0,0 +1,99 @@
+- Feature Name: abort_by_default
+- Start Date: 2016-10-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Specify abort-by-default in `Cargo.toml` when the user does `cargo new --bin`, as well as various other refinements to the panick strategy system.
+
+# Motivation
+[motivation]: #motivation
+
+## Performance
+
+Generally, the performance of bigger programs [improves by 10%](https://www.youtube.com/watch?v=mRGb4hoGuPs), which is no small amount. When overflow checking is enabled, it is as high as 2x, making overflowing checking plausible in production.
```

Comment:
https://www.reddit.com/r/rust/comments/55ns2m/safe_and_efficient_bidirectional_trees/ had a 20% cpu time improvement by disabling landing pads. There were two ways to reach that goal. One, to use the -Z option, second to eliminate all panics in the actual code.